### DISCUSSION TITLE
Can not run or debug anything from hover_actions

### DISCUSSION THREAD

**Discussion by cyejing** [simrat39/rust-tools.nvim#342] [Q&A]
![image](https://user-images.githubusercontent.com/6317049/222036236-bf6b46e4-d2f7-404e-a91b-ab3b01466099.png)
I start hover_actions with the cursor over a main. but the menu doesn't show `run` and `debug` option

`:RustRunnables` allows me to run the main.

who can help me. please

**Comment by 5iddy** [comment]
Was that supposed to happen? Hover action seems to only pull up documentation for whatever is under the cursor. You can just setup a keymap to run `RustRunnables`

```lua
vim.keymap.set("n", "<leader>rr", "<CMD>RustRunnables<CR>", { remap=false, silent=true, desc="Rust Runnables"})
```

**Reply by cyejing** [reply]
In earlier versions it was possible。
like this
![image](https://user-images.githubusercontent.com/6317049/222347619-e9bbdfed-eba4-4761-a3cd-a9b548b6465f.png)
